mocap_optitrack_driver/include/mocap_optitrack_driver/mocap_optitrack_driver.hpp

```diff
@@ -94,16 +105,28 @@ class OptitrackDriverNode : public mocap_control::ControlledLifecycleNode
   sFrameOfMocapData latest_data;
   sRigidBodyData latest_body_frame_data;
 
+  std::unordered_map<int, std::string> id_rigid_body_map;
+  std::unordered_map<std::string, int> rigid_body_id_map;
+  std::set<std::string> tf_frames_to_publish;
```

Comment:
Based on my understanding of the code, tf_frames_to_publish would be better named something like tf_rigid_bodies_to_publish.